workday.sgnl.yaml

```diff
@@ -0,0 +1,92 @@
+description: "Workday as a System of Record"
+address: "wd2-impl-services1.workday.com" 
```

Comment:
```suggestion
address: "{{Input Required}}" 
```

Reply:
@marc-sgnl is '"wd2-impl-services1.workday.com" ' a shared baseURI for all Workday instances?

Reply:
I do not believe so, I believe this varies across customers and geographies.